web-js/src/components/TodoItemsPage.js

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import {
+  Container,
+  TextField,
+  Button,
+  IconButton,
+  Typography,
+  Checkbox,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemSecondaryAction,
+  ListItemText,
+  withStyles,
+  LinearProgress,
+} from "@material-ui/core";
+import AddIcon from "@material-ui/icons/Add";
+import ClearIcon from "@material-ui/icons/Clear";
+import { useTodos } from "./useTodos";
+import { useDraftTodos } from "./useDraftTodos";
+import { useShowLoader } from "./util-hooks";
+import { MoreInfoTemplateAndDocs, MoreInfoGraphiQL } from './MoreInfo'
+
+export function TodoItemsPage() {
+  const { loading, todos, ...todoActions } = useTodos();
+  const { draftTodos, ...draftTodoActions } = useDraftTodos();
+  const showLoader = useShowLoader(loading, 200);
+  return (
+    <Container className="main-container" maxWidth="sm">
+      {loading ? (showLoader ? <LinearProgress /> : null) : (
+        <div className="todo-items-container">
+          <Typography component="p" variant="h5">
+            {`You have ${todos.length} To-Do Item${
+              todos.length === 1 ? "" : "s"
+            }`}
+          </Typography>
+          <Button
+            variant="contained"
+            color="primary"
+            startIcon={<AddIcon />}
+            onClick={() => draftTodoActions.createDraftTodo()}
+          >
+            Add To-Do
+          </Button>
+          <List style={{ width: "100%" }}>
+            {todos.map((todo) => (
+              <TodoItem
+                key={String(todo._id)}
+                todo={todo}
+                todoActions={todoActions}
+              />
+            ))}
+            {draftTodos.map((draft) => (
+              <DraftTodoItem
+                key={String(draft._id)}
+                todo={draft}
+                todoActions={todoActions}
+                draftTodoActions={draftTodoActions}
+              />
+            ))}
+          </List>
+        </div>
+      )}
+      <MoreInfoGraphiQL />
+      <MoreInfoTemplateAndDocs />
+    </Container>
+  );
+}
+
+function TodoItem({ isDraft = false, todo, todoActions }) {
```

Comment:
Did you mean to do something with the `isDraft` variable? It's not used anywhere here.

Reply:
Good catch - missed this in a refactor